.env.example

```diff
@@ -16,3 +16,4 @@ QUEUE_DRIVER=sync
 
 REDIS_HOST=127.0.0.1
 REDIS_PORT=6379
+HDX_URL=https://test-data.humdata.org/api/action
```

Comment:
@kinstelli I think it should just be https://test-data.humdata.org but I'm not sure.

Reply:
Not sure either. Maybe production is okay, since users still need a key.